functions_framework_tool/templates/cloudevent/README.md

```diff
@@ -0,0 +1,118 @@
+# Basic CloudEvent example
+
+This example demonstrates writing a function to handle a CloudEvent.
+
+CloudEvent function handlers don't return a response to send to the event
+producter. They generally perform some work and print output for logging.
+
+The basic shape of the function handler looks like this:
+
+```dart
+@CloudFunction()
+void function(CloudEvent event, RequestContext context) {
```

Comment:
This would be a functions_framework change (different package), the templates / examples just follow the framework. We've had quite a bit of discussion on the evolving shape of these handlers. But it's a good point! cc/ @kevmoo.

Reply:
Since the change is outside of the scope of this package, I'm going to resolve the conversation. The templates can be updated as the framework evolves.